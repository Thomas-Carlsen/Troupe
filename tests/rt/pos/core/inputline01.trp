let val _ = printWithLabels "What is your name?"
    val name = inputLine ()
in printWithLabels ("Hello", name) 
end
