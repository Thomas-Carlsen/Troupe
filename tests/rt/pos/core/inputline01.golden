2019-03-07T14:31:29.650Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
"What is your name?"@{}%{}
("Hello"@{}%{}, "Alice"@{#TOP}%{})@{}%{}
>>> Main thread finished with value: ()@{#TOP}%{#TOP}
